examples/react/webpack.config.js

```diff
@@ -36,8 +39,5 @@ module.exports = {
 
 	plugins: [
 		new HtmlWebpackPlugin(),
-		new ProvidePlugin({
-			React: 'react',
-		}),
```

Comment:
Let's keep this commented out and mention that says it's necessary for React versions that don't have `/jsx-runtime`